vhd/packer/windows-2019-vars.json

```diff
@@ -2,6 +2,6 @@
     "image_publisher": "MicrosoftWindowsServer",
     "image_offer": "WindowsServer",
     "image_sku": "2019-Datacenter-Core-smalldisk",
-    "image_version": "17763.2565.220202",
+    "image_version": "17763.3165.220706",
```

Comment:
Should we also update the `WindowsServer2019OSImageConfig` image version in pkg/api/azenvtypes.go?

Reply:
We usually do that after we build/publish VHDs